test/Microsoft.ML.TestFramework/Datasets.cs

```diff
@@ -160,6 +160,13 @@ public static class TestDatasets
             loaderSettings = "col=Label:R4:11 col=Features:R4:0-10 sep=; header+"
         };
 
+        public static TestDataset WikiDetox = new TestDataset
+        {
+            name = "WikiDetox",
+            trainFilename = "Input/WikiDetoxAnnotated160kRows.tsv",
```

Comment:
Should this be `"external/WikiDetoxAnnotated160kRows.tsv"?

Reply:
We are doing a similar thing for all the other files in the benchmark project https://github.com/dotnet/machinelearning/pull/820/files/d2fe71419e148aebd916626889caccb7fd25d845#diff-ea979ea573e3bccc8a3837b5e09743ffL28

Reply:
The difference here is that these Datasets are used throughout the tests, not just in the benchmark tests. 

I see you are mapping `..\data\external` to `Input\` in the `Benchmarks.csproj`.  However, other tests try to read these files from underneath `$RepoRoot/test/data`, and there is no "Input" folder in that directory.  Since this class holds common datasets used throughout the tests, I don't think it makes sense to have this "Input" directory in this file name in the common code.  If some other test tried using this dataset, it wouldn't work.